components/input.css

```diff
@@ -6,10 +6,14 @@
   border-radius: 4px;
 }
 .input {
-  padding: 5px;
-  margin: 1rem;
-  text-align: center;
+  padding: 5px 10px 5px 10px;
+  margin: 8px 1rem 8px 1rem;
```

Comment:
you can simply apply padding: 5px 10px; and margin: 8px 1rem; and it will work the same.

Reply:
Yeap ! changed to shorthand